Fix with big endian.

src/textdefs.c:45:20: error: expression is not assignable
   45 |                 RSHORT(&s->p[i]) = RSHORT(&TextBTable[(*buf) * 2]);
      |                 ~~~~~~~~~~~~~~~~ ^
1 error generated.

Index: src/textdefs.c
--- src/textdefs.c.orig
+++ src/textdefs.c
@@ -42,7 +42,8 @@ void str_hex(_WString* s, const uint8_t* buf, unsigned
 	s->length = len * 2;
 	s->p[len * 2] = 0;
 	do {
-		RSHORT(&s->p[i]) = RSHORT(&TextBTable[(*buf) * 2]);
+		s->p[i] = TextBTable[(*buf) * 2];
+		s->p[i + 1] = TextBTable[(*buf) * 2 + 1];
 		buf++;
 		i += 2;
 	} while (i < len * 2);
